### PR TITLE
Cottage14 patch 4 first draft of page on Q Bot

### PR DESCRIPTION
**Pull request by cottage14**

### CODE REVIEW

File: content/pages/qbot.md [hunk 1]
```diff
@@ -0,0 +1,55 @@
...
+  -  In the Slack interface, there is a general menu bar at the far left. The next column is the menu bar for the ASF workspace. At the bottom of the list of your contacts and channels, there is a section for `Apps`. Q Bot appears there as `Qbot`.
+        -  Click that entry to see a display with three tabs.
+        -  Select the `Messages` tab.
+        -  Write your command to Qbot without `/` or 'qbot` at the start.
+        
+
+## Q Bot commands
+Here is what you can do with Q Bot's commands. **Note** we are expanding Q Bot's reach, and this list will expand to match its capabilities.
+
+  - `addme`- provide the Slack ID of the person to add to the private channel. (Without Q Bot, you have to request in a Jira ticket that Infra do this.)
```
Comment (cwlls) [inline content/pages/qbot.md:44]: I believe this must be used by the person wishing to be added to the channel and has the form `addme <channelname>`

File: content/pages/qbot.md [hunk 2]
```diff
@@ -0,0 +1,55 @@
...
+  - `flip` - Q Bot will flip a coin and tell you whether it came up heads or tails.
+  - `help` - You will see a list of the available commands.
+  - `q` - This command starts a huddle in your current channel, and makes you the administrator. The syntax is `/qbot q start`.
```
Comment (cwlls) [inline content/pages/qbot.md:47]: This one is not so much about starting huddles, but rather about organizing a Q&A queue.

File: content/pages/qbot.md [hunk 3]
```diff
@@ -0,0 +1,55 @@
...
+
+It also has a set of 'fun' features, such as rolling dice or flipping a coin.
+
+## Setting up Q Bot in your PMC's channels
+If you want to have Q Bot available in your PMC's channel, create a Jira ticket for Infra with the request. Include this information:
+  - The name of the channel or channels where Q Bot notifications about Jira tickets related to your projects should appear
+  - How you want notifications to appear. There are five options:
+      - No notifications
+      - All notifications
+      - `nodescription`: All notifications without the title of the ticket, but not its description
```
Comment (cwlls) [inline content/pages/qbot.md:21]: **with** the title of the ticket, but not... (I think)

File: content/pages/qbot.md (hunk 1 above)
Comment (cottage14) [reply content/pages/qbot.md]: fixed

File: content/pages/qbot.md (hunk 2 above)
Comment (cottage14) [reply content/pages/qbot.md:47]: Huh. Oh, yes. The thing we have abandoned.

Comment (cottage14) [reply content/pages/qbot.md]: Fixed

File: content/pages/qbot.md (hunk 3 above)
Comment (cottage14) [reply content/pages/qbot.md]: Fixed

File: content/pages/qbot.md (hunk 1 above)
Comment (cwlls) [reply content/pages/qbot.md]: I think the description in parentheses needs tweaking. If Qbot isn't present, you have to request admittance from someone already in the channel.

Comment (cottage14) [reply content/pages/qbot.md]: tweaked

**Review by cwlls — APPROVED** [review]
LGTM +1

**Pull request merged**